src/teachers_gitlab/main.py

```diff
@@ -593,6 +593,56 @@ def action_clone(
         mg.clone_or_fetch(glb, project, local_path)
         mg.reset_to_commit(local_path, last_commit.id)
 
+@register_command('create-group', 'Create new group')
+def create_group(
+    glb: GitlabInstanceParameter(),
+    logger: LoggerParameter(),
+    entries: ActionEntriesParameter(),
+    group_name_template: ActionParameter(
+        'name',
+        required=False,
+        metavar='NAME_OF_THE_GROUP',
+        help='String name of the created group, formatted from CSV columns.'
```

Comment:
Maybe shorter help `Group name (title), formatted from CSV columns.` could be good enough?